src/components/themed/EdgeText.tsx

```diff
@@ -4,6 +4,8 @@ import { Platform, StyleProp, Text, TextProps, TextStyle } from 'react-native'
 import { fixSides, mapSides, sidesToPadding } from '../../util/sides'
 import { cacheStyles, Theme, useTheme } from '../services/ThemeContext'
 
+export type ParagraphFontType = 'default' | 'warning' | 'finePrint'
```

Comment:
Can't we do this the same way the Web platform handles it? Define components like `<BoldText>`, `<WarningText>`, `<FinePrint>`, which can be used inline. Then it's:

```html
<Paragraph>
  <WarningText>
    Do not lose your keys, or you will <BoldText>not</BoldText> be able to get them back!
  </WarningText>
</Paragraph>
```

Your `ParagraphFontType` way might be OK too, especially if we really lock down what combinations are valid or not, but it could also end up growing out of control. It's hard to predict how these things play out.

Reply:
Yeah I was struggling trying to define the rules and wasn't aware the suggested was possible. Works way better, thanks